Importer/CsvImporter.php

```diff
@@ -45,16 +45,16 @@ public function __construct(
     /**
      * @param string $filename
      * @param bool   $hasHeaders
-     * @param bool   $flush
+     * @param bool   $dryRun
      *
      * @return array|null|object[]
      * @throws DatabaseException
      */
-    public function importFile($filename, $hasHeaders = true, $flush = true)
+    public function importFile($filename, $hasHeaders = true, $dryRun = true)
```

Comment:
this should probably be false by default. It is a more sensible default and could also reduce the impact of this *large* BC break.